recipe/meta.yaml

```diff
@@ -30,15 +34,27 @@ test:
     - hatch_jupyter_builder
   commands:
     - pip check
-    - pip install -e ".[test]"
-    - python -m pytest -vv
+# Note windows tests supressed because of problems with utils.run(shlex.split(cmd))
+#    - python -m pytest -k "not (test_run or test_normalize_cmd)" -vv  # [win]
+    - pip install -e ".[test]" # [not (win or s390x)]
+    - python -m pytest -vv     # [not (win or s390x)]
   requires:
+    - hatchling
     - pip
+    - pytest               # [ not win]
+    - pytest-cov           # [ not win]
+    - pytest-mock          # [ not win]
+    - tomli                # [ not win]
+    - twine                # [ not win]
```

Comment:
I don't think twine is required by default, unless pytest is run with the `--migration-tests` flag (https://github.com/jupyterlab/hatch-jupyter-builder/blob/74d37d9854668fea5c51b57622aa12fc72f0a61c/tests/conftest.py#L7).